notebooks/basics/single-systems.ipynb

```diff
@@ -88,26 +85,26 @@
+    "The archetypal example, which we will come back to repeatedly, is that of a *bit*, which is a system whose classical states are 0 and 1. Any system with two states provides a realization of a classical bit: the labels are unimportant.\n",
     "Other examples include a standard six-sided die, whose classical states are 1, 2, 3, 4, 5, and 6; a nucleobase in a strand of DNA, whose classical states are *A*, *C*, *G*, and *T*; and a switch on an electric fan, whose classical states are (commonly) *high*, *medium*, *low*, and *off*.\n",
     "In mathematical terms, the specification of the classical states of a system are, in fact, the starting point: we define a bit to be a system that has classical states 0 and 1, and likewise for systems having different classical state sets.\n",
     "\n",
     "For the sake of this discussion, let us give the name $\\mathsf{X}$ to the system being considered, and let us use the symbol $\\class{_sigma}{\\Sigma}$ to refer to the set of classical states of $\\mathsf{X}$.\n",
-    "As well as the assumption that $\\Sigma$ is finite, as mentioned earlier, we naturally assume that $\\Sigma$ is nonempty: it is nonsensical for a physical system to have no states at all that it can be in.\n",
-    "Although it does make sense to consider physical systems having infinitely many classical states, we will ignore this possibility for now.\n",
+    "As well as the assumption that $\\Sigma$ is finite, as mentioned earlier, we naturally assume that $\\Sigma$ is nonempty: it is nonsensical for a physical system to have no states at all.\n",
+    "Although it does make sense to consider physical systems having infinitely many classical states, we will ignore this possibility for now. There are plenty of interesting ideas to explore even within only a finite number of states.\n",
     "For the sake of convenience and brevity, we will hereafter use the term *classical state set* to mean any finite and nonempty set.\n",
     "\n",
     "For example, if $\\mathsf{X}$ is a bit, then $\\Sigma = \\{0,1\\}$ (which is often called the *binary alphabet*);\n",
     "if $\\mathsf{X}$ is a six-sided die, then $\\Sigma = \\{1,2,3,4,5,6\\}$; \n",
     "and if $\\mathsf{X}$ is an electric fan switch, then \n",
     "$\\Sigma = \\{\\mathrm{high}, \\mathrm{medium}, \\mathrm{low}, \\mathrm{off}\\}$.\n",
     "\n",
-    "When thinking about $\\mathsf{X}$ as a carrier of information, where the different classical states of $\\mathsf{X}$ could have different interpretations or lead to different outcomes or consequences, it may be sufficient to describe $\\mathsf{X}$ as simply being in one of its possible classical states.\n",
+    "When thinking about $\\mathsf{X}$ as a carrier of information, the different classical states of $\\mathsf{X}$ will be assigned certain meanings, leading to different outcomes or consequences. In such cases, it may be sufficient to describe $\\mathsf{X}$ as simply being in one of its possible classical states.\n",
```

Comment:
I like the rewrite in general, but still prefer the word "could" to the word "will." The reason is that _we_ might not assign meanings in the text, so people shouldn't expect that this will happen.